core/templates/components/question-directives/question-misconception-selector/question-misconception-selector.component.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 The Oppia Authors. All Rights Reserved.
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
Change not required. Only required when creating new files.

Reply:
Done.

Reply:
Done.